tasklauncher-cloudfoundry-app-dependencies/pom.xml

```diff
@@ -13,15 +13,15 @@
     <parent>
         <artifactId>spring-cloud-dependencies-parent</artifactId>
         <groupId>org.springframework.cloud</groupId>
-        <version>1.2.1.RELEASE</version>
+        <version>1.2.1.BUILD-SNAPSHOT</version>
```

Comment:
@cppwfs What do you think about merging without the `1.3.x` upgrade as we can update it across all the projects if we want to do so?